applications/plugins/SofaOpenCL/CMakeLists.txt

```diff
@@ -94,6 +94,7 @@ endif()
 
 find_package(OpenCL REQUIRED)
 find_package(SofaGeneral REQUIRED)
+find_package(SofaSphFluid REQUIRED)
```

Comment:
Are you sure about the REQUIRED here? The rest of this CMakeLists makes me think the dependency to SofaSphFluid is soft.